src/Components/CandidateTable/fakeData.ts

```diff
@@ -3,39 +3,17 @@ import { Candidate, CandidateTable, Company, FilterProperty, Gender, Rank, User,
 // -------- fake tables data --------- //
 
 export const fakeUserData: User[] = [
-  // {
-  //   id: 0,
-  //   hashName: "",
-
-  //   nick: "",
-  //   avatarImage: "",
-  //   firstName: "",
-  //   lastName: "",
-  //   emailAdress: "",
-  //   gender: Gender.Other,
-  //   birthday: "",
-  //   country: "",
-  //   adress: "",
-  //   phoneNumber: "",
-  //   languages: [""],
-  //   programLanguages: [""],
-  //   status: UserStatus.NotSelected,
-
-  //   taskInProgress: [],
-  //   taskComplited: [],
-  //   taskNotFinished: [],
-  // },
   {
     id: 1,
-    hashName: "e0cf413338a0c044f835ac0a0990824eb1b9c204a813dfef411704f4949504cc",
+    hash: "e0cf413338a0c044f835ac0a0990824eb1b9c204a813dfef411704f4949504cc",
 
     nick: "CToshi",
     avatarImage: "https://www.publicdomainpictures.net/pictures/280000/nahled/not-found-image-15383864787lu.jpg",
     firstName: "Cort",
     lastName: "Tosh",
     emailAdress: "ctosh0@github.com",
     gender: Gender.Male,
-    birthday: "October 13, 1956",
+    birthday: new Date("October 13, 1956"),
```

Comment:
Oh wow, I didn't know that `Date` constructor can do this!
I think it's very readable. Good job. 🙇 

I hope it's also serialisable enough though... Reason is: you'll be getting this data over the wire from the backend, so you'd better have your mock data be exactly the same as the one you'd get from there. I'm not sure it's possible to send a JS Date object with plain JSON.

Don't change it yet, just keep that in mind.